cfme/tests/test_utilization.py

```diff
@@ -21,7 +21,7 @@
 # test_utilization_metrics.py go a step further and verify that specific performance metrics are
 # being collected.Eventually, support should be added to verify that specific metrics are being
 # collected for *all* providers.
-pytest_generate_tests = testgen.generate(testgen.provider_by_type, ['cloud',
+pytest_generate_tests = testgen.generate(testgen.providers_by_category, ['cloud',
```

Comment:
See, here, _IF_ we did stop using the strings(and I can see why you wouldn't want to with all these changes,) this could be a single ```testgen.providers_by_class()```